demo/full/styles/style.css

```diff
@@ -341,38 +439,155 @@ body {
   margin-top: -7px;
 }
 
+.black-knob select {
+  /* styling */
+  background-color: #000;
+  display: inline-block;
+  font: inherit;
+
+  margin: 0;
+  padding: 3px 21px 3px 3px;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+
+  background-image: url("../assets/select_arrow_white.svg");
+  background-size: 8px;
+  background-repeat: no-repeat;
+  background-position: calc(100% - 8px) 50%;
+}
+
+.black-knob select:disabled {
+  -moz-appearance:none;
+  -webkit-appearance:none;
+  appearance:none;
+  background-image: none;
+}
+
+.player-knobs-header {
+  padding: 9px 20px 9px 20px;
+  border-bottom: 1px dotted #555;
+}
+
 .player-knobs {
-  margin: 0px auto 20px auto;
-  text-align: left;
+  width: 60%;
+  max-width: 380px;
+  min-width: 280px;
+  color: white;
+  position: absolute;
+  bottom: 10px;
+  right: 10px;
+  background-color: rgba(0,0,0);
+  opacity: 0;
+  transform: scale(0.9);
+  transition: opacity 200ms;
+  z-index: 100;
+  pointer-events:none;
+}
+
+.player-knobs.fade-in-out {
+  transform: scale(1);
+  pointer-events: auto;
+  opacity: 0.8;
+}
+
+.player-knobs-title {
+  font-weight: bold;
+}
+
+.player-knobs-close {
+  font-family: "icons", sans-serif;
+  color: #d0d0cf;
+  font-size: 14px;
+  margin-top: auto;
+  margin-bottom: auto;
+  cursor: pointer;
+}
+
+.player-knobs-close:hover {
+  color: #ffffff;
+}
+
+.player-knobs-knobs {
+  padding: 10px 20px 10px 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.player-knobs > div {
+  display: flex;
+  justify-content: space-between;
 }
 
 .knob {
-  display: inline-block;
-  margin: 10px;
+  display: flex;
+  margin: 5px 0 5px 0;
 }
 
 .knob-name {
-  display: inline-block;
-  margin-right: 10px;
+  margin: auto 10px auto auto;
+  overflow: hidden;
+  white-space: nowrap;
+  flex: 1;
 }
 
 .knob-value {
   display: inline-block;
+  flex: 1;
+}
+
+.knob-value > select:disabled {
+  color: rgb(130,130,130);
+}
+
+.knob-value > select {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Expected selector ".knob-value > select" to come before selector ".knob-value > select:disabled" ([css:S4664](https://sonarcloud.io/organizations/rx-player/rules#rule_key=css%3AS4664))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=312&issues=AWXYzQyS_eEZPtUmFEz4&open=AWXYzQyS_eEZPtUmFEz4)

[ISSUE_KEY]: AWXYzQyS_eEZPtUmFEz4